spec/integration/store/TimelineSpace/Modals/Jump.spec.js

```diff
@@ -4,56 +4,59 @@ import i18n from '~/src/config/i18n'
 import router from '@/router'
 import Jump from '~/src/renderer/store/TimelineSpace/Modals/Jump'
 
-const state = {
-  modalOpen: true,
-  channel: '',
-  defaultChannelList: [
-    {
+const state= () => {
```

Comment:
__[ESLint]__ Error - Infix operators must be spaced. (space-infix-ops)

<sub>posted by Sider</sub>